src/index.js

```diff
@@ -0,0 +1,79 @@
+// import things
```

Comment:
I think it's really awesome that you mapped out your thought process and organized your code so well - you have really clean code. However, starting sometime next week, we're going to start taking points off for leaving comments in your code - even though lines like this are really helpful for outlining while you're working on code, you will ultimately want to delete any comments before submitting your labs. If you want to save the comments, just make sure you commit before you delete them and then you can refer back to that commit to have the comment history. Other than that, this lab looks great!